media/js/firefox/new/experiment-win64-funnelcakes-scene2.js

```diff
@@ -0,0 +1,15 @@
+(function() {
+    'use strict';
+
+    // update dl link for windows to point to win64
+    // same selector from canonical scene2.js
+    var $platformLink = $('#download-button-wrapper-desktop .download-list li:visible .download-link');
```

Comment:
I thought about that, but this is the selector used in `scene2.js`, which is used to trigger the download (and populate the 'click here' link). On one hand you're making sure you hit the link actually used, and on the other the specific link for `.os_win`.